src/test/java/com/github/stickerifier/stickerify/media/MediaHelperTest.java

```diff
@@ -127,67 +125,147 @@ private void assertVideoConsistency(int expectedWidth, int expectedHeight, float
+		var result = MediaHelper.convert(startingVideo);
+
+		assertVideoConsistency(result, 512, 288, 29.97F, 3_000L);
+	}
+
 	@Test
 	void convertShortAndLowFpsVideo() throws Exception {
 		var startingVideo = resources.loadResource("short_low_fps.webm");
-		result = MediaHelper.convert(startingVideo);
+		var result = MediaHelper.convert(startingVideo);
 
-		assertVideoConsistency(512, 288, 10F, 1_000L);
+		assertVideoConsistency(result, 512, 288, 10F, 1_000L);
 	}
 
 	@Test
 	void resizeSmallWebmVideo() throws Exception {
 		var startingVideo = resources.loadResource("small_video_sticker.webm");
-		result = MediaHelper.convert(startingVideo);
+		var result = MediaHelper.convert(startingVideo);
 
-		assertVideoConsistency(512, 212, 30F, 2_000L);
+		assertVideoConsistency(result, 512, 212, 30F, 2_000L);
 	}
 
 	@Test
 	void convertVerticalWebmVideo() throws Exception {
 		var startingVideo = resources.loadResource("vertical_video_sticker.webm");
-		result = MediaHelper.convert(startingVideo);
+		var result = MediaHelper.convert(startingVideo);
 
-		assertVideoConsistency(288, 512, 30F, 2_000L);
+		assertVideoConsistency(result, 288, 512, 30F, 2_000L);
 	}
 
 	@Test
 	void convertGifVideo() throws Exception {
 		var startingVideo = resources.loadResource("valid.gif");
-		result = MediaHelper.convert(startingVideo);
+		var result = MediaHelper.convert(startingVideo);
 
-		assertVideoConsistency(512, 274, 10F, 1_000L);
+		assertVideoConsistency(result, 512, 274, 10F, 1_000L);
 	}
 
 	@Test
 	void noVideoConversionNeeded() throws Exception {
 		var startingVideo = resources.loadResource("no_conversion_needed.webm");
-		result = MediaHelper.convert(startingVideo);
+		var result = MediaHelper.convert(startingVideo);
 
 		assertThat(result, is(nullValue()));
 	}
 
 	@Test
 	void noAnimatedStickerConversionNeeded() throws Exception {
 		var animatedSticker = resources.loadResource("animated_sticker.gz");
-		result = MediaHelper.convert(animatedSticker);
+		var result = MediaHelper.convert(animatedSticker);
 
 		assertThat(result, is(nullValue()));
 	}
 
 	@Test
 	void unsupportedGzipArchive() {
 		var archive = resources.loadResource("unsupported_archive.gz");
-		TelegramApiException exception = assertThrows(TelegramApiException.class, () -> MediaHelper.convert(archive));
 
-		assertThat(exception.getMessage(), is(equalTo("Passed-in file is not supported")));
+		assertThrows(TelegramApiException.class, () -> MediaHelper.convert(archive));
 	}
 
 	@Test
 	void unsupportedFile() {
 		var document = resources.loadResource("document.txt");
-		TelegramApiException exception = assertThrows(TelegramApiException.class, () -> MediaHelper.convert(document));
 
-		assertThat(exception.getMessage(), is(equalTo("Passed-in file is not supported")));
+		assertThrows(TelegramApiException.class, () -> MediaHelper.convert(document));
+	}
+
+	@Nested
+	@DisplayName("Concurrently convert")
+	@DisabledIfEnvironmentVariable(named = "EXECUTION_ENVIRONMENT", matches = "local")
```

Comment:
If your intention is to only run these tests on Github Workflow, you could use the `CI` env variable, which is already present.